src/org/exist/storage/BrokerPool.java

```diff
@@ -871,6 +870,16 @@ protected void initialize() throws EXistException, DatabaseConfigurationExceptio
                         symbols = new SymbolTable(conf);
                         isReadOnly = isReadOnly || !symbols.getFile().canWrite();
 
+                        try {
+                            // initialize EXPath repository so indexManager and
+                            // startup triggers can access it
+                            expathRepo = ExistRepository.getRepository(this.conf);
+                        } catch(final PackageException e) {
+                            LOG.warn("Failed to initialize expath repository: " + e.getMessage() + " - " +
```

Comment:
It seems to me that his should be an error and not a warning. As the package repository will be absolutely unavailable if an exception is thrown here... Not to mention that subsequent attempted access will likely cause an NPE.